From e23752d5b12df62113d0a63e58de066916a59c40 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Fri, 5 Dec 2014 13:56:54 +0800
Subject: [PATCH 0343/1594] MLK-9962-2: ARM: dts: imx6sx-sabreauto: change
 asrc_p2p to asrc

commit e9d1cf7bc3ea3dc2c6ee5d7d27ff752ad0b7bd49 from
git://git.freescale.com/imx/linux-2.6-imx.git

In 3.14, the asrc_p2p and asrc node has been merged together. So
the asrc_p2p need to be changed to asrc.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 5c02d50..40a0cff 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -177,7 +177,7 @@
 				"fsl,imx-audio-cs42888";
 		model = "imx-cs42888";
 		esai-controller = <&esai>;
-		asrc-controller = <&asrc_p2p>;
+		asrc-controller = <&asrc>;
 		audio-codec = <&codec>;
 	};
 
-- 
1.7.5.4

